src/components/EditCoursePage/index.jsx

```diff
@@ -126,6 +127,7 @@ class EditCoursePage extends React.Component {
     const {
       editCourse,
     } = this.props;
+
     // Create a deep copy of course runs since we modify their properties
     let modifiedCourseRuns = jsonDeepCopy(courseData.course_runs);
```

Comment:
I don't have strong preferences on which way you choose, but I'd prefer consistency between prepareCourseRuns and prepareCourse. Like we are doing the jsonDeepCopy outside and then letting prepareCourseRuns modify the modifiedCourseRuns, but we do the opposite in prepareCourse.

Reply:
@Dillon-Dumesnil I made updates for this request, please review when you have time.